node/src/components/storage.rs

```diff
@@ -2384,4 +2384,25 @@ impl Storage {
             execution_result,
         })
     }
+
+    pub(crate) fn delete_block_utilization_score_by_block_hash(&mut self, block_hash: BlockHash) {
```

Comment:
should this have the test cfg attribute?

Reply:
So the storage component has a separate test only impl, and its in that impl

```
#[cfg(test)] 
impl Storage {
 }
 ````